trieste/acquisition/optimizer.py

```diff
@@ -140,3 +143,44 @@ def target_func_with_vectorized_inputs(
         return tf.reshape(vectorized_points, [batch_size, -1])  # [B, D]
 
     return optimizer
+
+
+def batchify_greedy(
```

Comment:
does it make sense to rename `batchify` to explain how it differs from this kind of batching?